tsconfig.json

```diff
@@ -0,0 +1,62 @@
+{
+  "include": [
+    "./src/*.ts",
+    "./src/**/*.ts",
+    "./custom_typings/*.d.ts"
```

Comment:
I think you can move this to typeRoots

Reply:
No more custom_typings needed; have dropped parse5-traverse since nodeWalkAll copied over from dom5 did the job I needed, I just didn't realize it at the time.